internal/conf/configuration.go

```diff
@@ -707,6 +709,23 @@ func (config *GlobalConfiguration) ApplyDefaults() error {
 		config.JWT.Exp = 3600
 	}
 
+	if config.JWT.Keys == nil {
+		if config.JWT.KeyID != "" && config.JWT.Secret != "" {
+			config.JWT.Keys = make(JwtKeysDecoder)
+			derBytes, err := base64.StdEncoding.DecodeString(config.JWT.Secret)
+			if err != nil {
+				derBytes = []byte(config.JWT.Secret)
+			}
+			privKey, err := jwk.FromRaw(derBytes)
+			if err != nil {
+				return err
+			}
+			config.JWT.Keys[config.JWT.KeyID] = JwkInfo{
+				PrivateKey: privKey,
+			}
+		}
```

Comment:
Hm this feels wrong. DER is an encoding standard for private keys, but `secret` is likely to only just be a symmetric key in which case it's not DER?

Reply:
The `jwk.FromRaw` [method](https://pkg.go.dev/github.com/lestrrat-go/jwx/v2/jwk#FromRaw) detects if the key passed in is a symmetric or asymmetric key based on the type. If it's a `[]byte`, it assumes that the key is a symmetric key. The naming used here is incorrect though, it shouldn't be called `derBytes`.